docs/sources/writing-guide/reuse-shared-content.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
General comment. The writers' toolkit repo uses only _index.md or index.md files for content.

Reply:
Our latest recommendation in Writers' Toolkit is to prefer pages unless bundles are needed: https://grafana.com/docs/writers-toolkit/writing-guide/documentation-structure/#:~:text=If%20you%20don%E2%80%99t%20know%20whether%20you%20need%20to%20mount%20a%20leaf%20bundle%2C%20you%20probably%20don%E2%80%99t%20and%20default%20to%20using%20pages.